### ISSUE TITLE
How does Hunter compare to fips?

### ISSUE THREAD

**Issue by LegalizeAdulthood** [ruslo/hunter#178]
Just stumbled across [fips](http://floooh.github.io/fips/index.html) today.

Making C++ packaging systems seems to be all the rage these days....

However, instead of N people making N systems, I'd really like to see N people collaborate on making one great system.

biicode is gaining ground because they've been getting lots of exposure in media.  How/if that will translate into a workable system is unclear.  Meanwhile, it seems everyone and his brother is coding their own "one man show" of a packaging/build system.

Who will win?
##

**Comment by aadityakalsi** [comment]
I doubt that I can answer the who will win question. I can only answer why I contribute here and prefer this tool. In some sense, we contribute where we feel things are how they ought to be (or at least I do).

Hunter is almost purely CMake based. It doesn't require you to learn directory structures of your own projects like biicode or follow other tool specific things. There is obviously something to learn about using Hunter, but when I started I simply followed the first few things in the README and was on my way.

I found it easy to contribute, and get patches accepted, and I also found that I got fixes quickly. 

Everything that needs support was built-in and when I saw the ability to cross reference dependencies from within Hunter, like Boost needing ZLIB which is also in Hunter, and that all worked, that vetted the design very well for me.

In short, its low intrusion, easy, efficient and gets out of my way most of the time, rather than making me learn its way of doing things.

That help?

**Comment by pretyman** [comment]
Pretty much the same reason as Aaditya: I found it non-intrusive and easy to contribute. I only found hunter one month ago and did pretty much what I was looking for.

Ruslan, the owner of the project, when is not burning people's cars and doing the usual Russian debt collector stuff, is pretty nice and patient at giving advice, since I am a CMake newbie.

I only contribute with "bed-time development" as a hobby and it has been fun so far.

**Comment by ruslo** [comment]
> Just stumbled across fips today

@LegalizeAdulthood Thanks for the link. I'll take a look.

**Comment by ruslo** [comment]
Have no plans to write such articles. If you have one feel free to add it to the [F.A.Q.](https://docs.hunter.sh/en/latest/faq.html).
